.github/workflows/integration-test.yml

```diff
@@ -5,6 +5,10 @@ on: [push, pull_request]
 env:
   TEST_BROWSER_HEADLESS: 1
   CI: 1
+  OPENSEARCH_DASHBOARDS_VERSION: 2.0
+  OPENSEARCH_CORE_VERSION: 2.0
+  OPENSEARCH_CORE_VERSION_QUALIFIER: ${{OPENSEARCH_CORE_VERSION}}.0
```

Comment:
nit: This doesn't look needed. Changing `OPENSEARCH_DASHBOARDS_VERSION` and `OPENSEARCH_CORE_VERSION` to 2.0.0 will be sufficient.